<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Update_cab_status</class>
 <widget class="QDialog" name="Update_cab_status">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Feedback</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QRadioButton{
    
	color: rgb(74, 3, 255);
}
QRadioButton::hover{
   color:black;
}
QRadioButton::pressed{
   color:black;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="img_rating">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_yesno">
     <item>
      <widget class="QLabel" name="label_haveureached">
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Have you reached your destination?</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_Yes">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QRadioButton{
}</string>
       </property>
       <property name="text">
        <string>Yes</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_No">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>No</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_enter_cabid">
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Enter the cab id you've travelled with</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit_ente_cab_id"/>
   </item>
   <item>
    <widget class="QLabel" name="label_rate">
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Rate your experience</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_stars">
     <item>
      <widget class="QRadioButton" name="radioButton_1star">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>1 star</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_2star">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>2 stars</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_3star">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>3 stars</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_4star">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>4 stars</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_5star">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>5 stars</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_ucs_back">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
  background-color: rgb(255, 134, 162);
  color: white;
  border-radius: 5px;
  padding: 5px;
  font-weight: bold;
  border: 2px solid black;
}

QPushButton:hover {
  background-color: rgb(166, 98, 200);
  font-style: italic;
  border: 2px solid white;
  color:black;
}

QPushButton:pressed {
  background-color: #008080;
  font-weight: bold;
  color: white;
  border: 2px solid white;
}
</string>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_proceed">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
  background-color: #1E90FF;
  color: white;
  border-radius: 5px;
  padding: 5px;
  font-weight: bold;
  border: 2px solid black;
}

QPushButton:hover {
  background-color: rgb(166, 98, 200);
  font-style: italic;
  border: 2px solid white;
  color:black;
}

QPushButton:pressed {
  background-color: #008080;
  font-weight: bold;
  color: white;
  border: 2px solid white;
}
</string>
       </property>
       <property name="text">
        <string>Proceed</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_submit">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
  background-color: #1E90FF;
  color: white;
  border-radius: 5px;
  padding: 5px;
  font-weight: bold;
  border: 2px solid black;
}

QPushButton:hover {
  background-color: rgb(166, 98, 200);
  font-style: italic;
  border: 2px solid white;
  color:black;
}

QPushButton:pressed {
  background-color: #008080;
  font-weight: bold;
  color: white;
  border: 2px solid white;
}
</string>
       </property>
       <property name="text">
        <string>Submit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
